src/main/webapp/app/shared/pages/privacy-page/privacy-page.component.scss

```
@@ -0,0 +1,54 @@
+.privacy-page {
+  color: var(--p-text-primary);
+  padding: 3rem 5rem 1.5rem 1.5rem;
+  max-width: 1200px;
+  margin-right: auto;
+
+  .headline {
+    font-size: 2.3rem;
```

Please use 2rem here as well
```suggestion
    font-size: 2rem;
```